dist/spec.json

```diff
@@ -13,7 +13,7 @@
       "name": "Apache-2.0",
       "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
     },
-    "version": "2.2.1"
```

Comment:
@bretterer Any idea why `2.2.1` shows up here when the latest is `2.2.4` presently?